WorkingGames.md

```diff
@@ -19,20 +21,26 @@ List of working and non-working games. Games not on this list have not been test
 
 ## Non-working games
 
+- Any Unreal Engine 4 game (the engine doesn't support resolution changes through OpenVR calls)
```

Comment:
```suggestion
- Any Unreal Engine game (the engine doesn't support resolution changes through OpenVR calls)
```
If UE5 don’t support them either, let’s just do that

Reply:
yeah. plus, all the UE5 games I know of I can't even test because I'm on Quest 2, and Unreal Engine 5 doesn't support OpenXR on SteamVR runtime through Quest 2, only Oculus runtime.